src/app-layer-smb.c

```diff
@@ -98,160 +98,132 @@ static uint32_t SMBParseWriteAndX(Flow *f, void *smb_state,
                 if (!(--input_len))
                     break;
             }
-            /* fall through */
```

Comment:
why would you remove this? it's important to tell devs AND code checkers that we fall through intentionally